network/evpn-gw/evpn-gw-cni/pkg/xpu/xpu.go

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Nordix Foundation.
+
```

Comment:
should use client code from here https://github.com/opiproject/godpu/tree/main/network instead of this duplication